lighthouse-core/lib/styles-helpers.js

```diff
@@ -104,29 +104,25 @@ ${parsedContent.selector} {
 }
 
 /**
- * Returns a array of all CSS prefixes and the default CSS style names.
+ * Returns an array of all CSS prefixes and the default CSS style names.
  *
- * @param {string|Array<string>=} propNames CSS text content.
```

Comment:
This should strictly be an array: `{Array<string>=}`.

Your current implementation wouldn't work if it were passed a string.